addons/point_of_sale/static/src/app/screens/partner_list/partner_editor/partner_editor.xml

```diff
@@ -48,18 +48,18 @@
                     <div class="partner-detail col">
                         <label class="form-label label" for="language">Language</label>
                         <select class="detail form-select" id="language" name="lang" t-model="changes.lang" t-att-class="{'border-danger': missingFields.includes('lang')}">
-                            <t t-foreach="pos.langs" t-as="lang" t-key="lang.id">
-                                <option t-att-value="lang.code" t-att-selected="changes.lang ? ((lang.code === changes.lang) ? true : undefined) : lang.code === pos.user.lang? true : undefined">
+                            <t t-foreach="pos.models['res.lang'].getAll()" t-as="lang" t-key="lang.id">
+                                <option t-att-value="lang.code" t-att-selected="changes.lang ? ((lang.code === changes.lang) ? true : undefined) : lang.code === pos['res.users'].lang? true : undefined">
                                     <t t-esc="lang.name" />
                                 </option>
                             </t>
                         </select>
                     </div>
-                    <div t-if="pos.pricelists.length gt 1" class="partner-detail col">
+                    <div t-if="pos.models['product.pricelist'].length gt 1" class="partner-detail col">
```

Comment:
```suggestion
                    <div t-if="pos.models['product.pricelist'].getLength() gt 1" class="partner-detail col">
```

Introduce a method in the related models module called `getLength()` (or similar name) to get the number of records.

Reply:
In related models we use classic proto from array like map, reduce, filter etc. It will be weird to not have the .length one, even if isn't really a proto

Reply:
But if you want I can do an alias

Reply:
Oops, sorry. I just saw that you already introduced `get length()`. Ignore this.